pkg/service/test/test.go

```diff
@@ -867,8 +870,10 @@ func (t *tester) testHttp(tc models.TestCase, actualResponse *models.HttpResp, n
 
 	if !pass {
 		logDiffs := NewDiffsPrinter(tc.Name)
-
 		logger := pp.New()
+		if !*EnableANSIColor {
```

Comment:
why are you using a pointer here?

Reply:
The reason for using a pointer is that we are passing a pointer to the const enableANSIColor which holds the value of the flag ,a pointer to which is being passed down to this function, instead of passing the value.

Reply:
But if it is a const variables why do you need to pass it by reference? you are not going to change the value inside the function, which is what leads to duplication, you are just going to use it right?

Reply:
Will address this once the cmd layer is refactored and we can move ahead with this pull request